### PR TITLE
Github app instructions

### PR DESCRIPTION
**Pull request by schottj**
Rewriting the configuring Github app instructions for dbt Cloud

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-installing-the-github-application.md [hunk 1]
```diff
@@ -1,43 +1,57 @@
 ---
-title: "Connecting your GitHub Account"
+title: "Connecting your GitHub account"
 id: "cloud-installing-the-github-application"
 sidebar_label: "Connecting GitHub"
 ---
 
 ## Overview
 
 Connecting your GitHub account to dbt Cloud unlocks exciting and compelling functionality in dbt Cloud. Once your GitHub account is connected, you can:
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-installing-the-github-application.md]:
```suggestion
Connecting your GitHub account to dbt Cloud provides convenience and another layer of security to dbt Cloud:
```

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-installing-the-github-application.md [hunk 2]
```diff
@@ -1,43 +1,57 @@
...
-- [Trigger CI builds](cloud-enabling-continuous-integration-with-github) when Pull Requests are opened in GitHub
-- Log into dbt Cloud via OAuth through GitHub
-- Add new repos to Cloud in a single click (no need to fuss with Deploy Keys)
+- Log into dbt Cloud via OAuth through GitHub.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-installing-the-github-application.md]:
```suggestion
- Log into dbt Cloud using OAuth through GitHub.
```

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-installing-the-github-application.md [hunk 3]
```diff
@@ -1,43 +1,57 @@
...
+docs on [importing a project by git URL](docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-import-a-project-by-git-url#github) to set up your connection. This alternative connection method does not include the benefits of the native integration mentioned above.
 :::
 
-## Linking dbt Cloud to your GitHub account
+## Installing dbt Cloud in your GitHub account
+
+A GitHub organization owner needs to connect and configure the dbt Cloud app for their GitHub organization. If you are a team of one or wish to connect your personal GitHub account, then these instructions also apply to you.
 
-To link your dbt Cloud account to your GitHub account, navigate to your [user profile](https://cloud.getdbt.com/#/profile/) and click the [Integrations section](https://cloud.getdbt.com/#/profile/integrations/). Here you can link your GitHub account to dbt Cloud.
+To link your dbt Cloud account to your team's GitHub account, navigate to your settings and scroll to **Linked Accounts** under **Your Profile**.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-installing-the-github-application.md]:
```suggestion
To link your dbt Cloud account to your team's GitHub account, navigate to Your Profile settings by clicking the gear icon in the top right. Select **Linked Accounts** in the left menu. 
```

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-installing-the-github-application.md [hunk 4]
```diff
@@ -1,43 +1,57 @@
...
+To link your dbt Cloud account to your team's GitHub account, navigate to your settings and scroll to **Linked Accounts** under **Your Profile**. 
 
-<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/profile_integrations_page.png" title="Click the green button to connect dbt Cloud to your GitHub account"/>
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-github/github-connect.gif" title="Navigated to Linked Accounts under your profile"/>
 
-On the next page, you can select a GitHub organization to install the dbt Cloud application into.
+Once in this section, you'll be able to set up your GitHub account connection to dbt Cloud by clicking on the **Link** button for GitHub. This will redirect you to your GitHub account where you will be asked to install and configure the dbt Cloud application. Select the GitHub organization you want dbt Cloud to have access to and which repositories you wish to grant access to.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-installing-the-github-application.md]:
```suggestion
In the Linked Accounts section, you can set up your GitHub account connection to dbt Cloud by clicking **Link** to the right of GitHub. This redirects you to your account on GitHub where you will be asked to install and configure the dbt Cloud application. Select the GitHub organization and repositories dbt Cloud should access.
```

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-installing-the-github-application.md [hunk 5]
```diff
@@ -1,43 +1,57 @@
...
 The dbt Cloud GitHub App requires the following permissions:
 - Read access to metadata
-- Read and write access to checks, code, commit statuses, pull requests, and workflows (new!)
+- Read and write access to checks, code, commit statuses, pull requests, and workflows
+
+Once you grant access to the app, you will be redirected back to dbt Cloud and shown a linked account success state. You are now personally authenticated too, and your team members can begin [connecting their profiles](/cloud-installing-the-github-application#personally-authenticate-with-github).
 
-## Usage notes
+## Configuring the dbt Cloud app in your GitHub account
+If you are your GitHub organization owner, you can also configure the dbt Cloud GitHub application to have access to only select repositories. This configuration must be done in GitHub, but we provide an easy link to start the process in dbt Cloud.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-installing-the-github-application.md]:
```suggestion
If you are your GitHub organization owner, you can also configure the dbt Cloud GitHub application to have access to only select repositories. This configuration must be done in GitHub, but we provide an easy link in dbt Cloud to start this process. 
```

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-installing-the-github-application.md [hunk 6]
```diff
@@ -1,43 +1,57 @@
...
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-github/configure-github.png" title="Configuring the dbt Cloud app"/>
 
-A GitHub organization owner needs to connect and configure the dbt Cloud app for their organization. Users on the Developer or Team plans do not need to each connect to GitHub, but it is recommended to do so. Users on the Enterprise plan must each connect their GitHub accounts, as dbt Cloud will enforce the repository's access permissions for every user in the IDE.
+## Personally authenticate with GitHub
+dbt Cloud developers on the Enterprise plan must each connect their GitHub profiles to dbt Cloud, as every developer's read / write access for the dbt repo is checked in the dbt Cloud IDE. dbt Cloud developers on the Team plan do not need to each connect their profiles to GitHub, but it is still recommended to do so. 
 
-## Connect your personal GitHub account
+To connect your personal GitHub account, dbt Cloud developers should navigate to their settings and scroll to **Linked Accounts** under **Your Profile**.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-installing-the-github-application.md]:
```suggestion
To connect a personal GitHub account, dbt Cloud developers should navigate to Your Profile settings by clicking the gear icon in the top right, then select **Linked Accounts** in the left menu.
```

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-installing-the-github-application.md [hunk 7]
```diff
@@ -1,43 +1,57 @@
...
 
-To connect your personal GitHub account, navigate to your [User Profile](https://cloud.getdbt.com/#/profile/) select the [Integrations section](https://cloud.getdbt.com/#/profile/integrations/) from the left sidebar. Under “GitHub”, if your account is already connected, you’ll see "Your user account is linked to a GitHub account with username &lt;your-github-username&gt;."
+If your GitHub account is not connected, you’ll see "No connected account". Select **Link** to begin the setup process. You’ll be redirected to GitHub, and asked to authorize dbt Cloud in a grant screen.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-installing-the-github-application.md:54]:
```suggestion
If your GitHub account is not connected, you’ll see "No connected account." Select **Link** to begin the setup process. You’ll be redirected to GitHub, and asked to authorize dbt Cloud in a grant screen.
```

**Review by runleonarun — APPROVED** [review]
@schottj Just a few edits.

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-installing-the-github-application.md [hunk 8]
```diff
@@ -1,43 +1,57 @@
...
 ---
 
 ## Overview
 
 Connecting your GitHub account to dbt Cloud unlocks exciting and compelling functionality in dbt Cloud. Once your GitHub account is connected, you can:
-- [Trigger CI builds](cloud-enabling-continuous-integration-with-github) when Pull Requests are opened in GitHub
-- Log into dbt Cloud via OAuth through GitHub
-- Add new repos to Cloud in a single click (no need to fuss with Deploy Keys)
+- Log into dbt Cloud via OAuth through GitHub.
+- Import new GitHub repos with a couple clicks during dbt Cloud project setup.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-installing-the-github-application.md]:
```suggestion
- Import new GitHub repositories with a couple clicks during dbt Cloud project setup.
```

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-installing-the-github-application.md (hunk 7 above)
Comment (schottj) [reply website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-installing-the-github-application.md:54]: I believe this is grammatically correct as is

**Pull request merged**